allure-java-commons/src/main/java/io/qameta/allure/Description.java

```diff
@@ -26,4 +26,12 @@
      * @return boolean flag to enable description extraction from javadoc.
      */
     boolean useJavaDoc() default false;
+
+    /**
+     * This flag adds <b>&#60;br /&#62;</b> tag after each line of javadoc description.
+     * Works only with {@link io.qameta.allure.Description#useJavaDoc()} enabled.
+     *
+     * @return boolean flag to enable line separation
+     */
+    boolean separateLines() default false;
```

Comment:
Is there any reason to have single line description? Seems like we can enable this feature for all users by default

Reply:
At first I had the same mind, but some users may use <b>&#60;br /&#62;</b> tags in their javadoc description manually, so this flag will duplicate their tags. But you have the last word, I will apply separation default if it's okay.

Reply:
> but some users may use <br /> tags in their javadoc description manually

hold on, I'll think about it

Reply:
@baev could you please give ETA of your decision?

Reply:
this Sunday

Reply:
ok, I feel like this should be a system property to configure such behaviour because you have the same javadocs formatting for whole project

Reply:
implementation notes:

* Lets call the property `allure.description.javadoc.separateLines`
*  you should be able to specify such property in `allure.properties` file (see https://github.com/allure-framework/allure-java/blob/master/allure-java-commons/src/main/java/io/qameta/allure/util/PropertiesUtils.java#L23)

Reply:
Please take a look